spec/fixtures/aws-lambda.lua

```diff
@@ -57,6 +57,9 @@ local fixtures = {
                     elseif string.match(ngx.var.uri, "functionEcho") then
                       require("spec.fixtures.mock_upstream").send_default_json_response()
 
+                    elseif string.match(ngx.var.uri, "functionWithTransferEncodingHeader") then
```

Comment:
This is not strictly AWS-related. I'd prefer to add it as a part of the regression test of PDK.

Reply:
Oops, forget to upload nginx test. Please check again, the original test case has a flaw which did not check if `transfer-encoding` exists